### PR TITLE
Rebalances Meteor Shields, more of them and faster deployment

### PR DESCRIPTION
**Pull request by mystery3525**
## About The Pull Request

Doubles the meteor shield satellite crate supply amount to 4 (totaling 28 shield satellites if all of them are deployed), and makes the deployment time 3 seconds (down from 10 seconds).

## Why It's Good For The Game

With https://github.com/BeeStation/BeeStation-Hornet/pull/9239 keyed, it is stated that meteor satellite deployment should be made easier, this makes it way more easier to rapidly acquire and deploy meteor satellites in the very limited amount of time we're forcing everyone to have to prepare. 10 seconds per satellite to deploy is ludicrous in my opinion, and is objectively worse than just dragging around a meteor crate from before. I have also added more supply to the meteor satellites, as it should not be as supply limited as other items due to the imminent emergency, no one is scalping the sats off the market.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Only cooldown and supply changes for now, if I need to prove this works in game then we aren't trusting the rest of the code to work.

</details>

## Changelog
:cl:
balance: Meteor shield satellites now take 3 seconds to deploy, down from 10.
balance: Meteor shield satellites now have double the supply, up to 4 packs of 28 total, from 2 packs of 14.
/:cl:

### CODE REVIEW

**Comment by EvilDragonfiend** [comment]
Although I introduced the feature, I don't care about the adjustment here. I will put this to other maints since I usually don't set meteor shields so that I don't know how ludicrous it is...

**Comment by Rukofamicom** [comment]
Since you have cherry-picked part of what was said on the other PR, let me slap both of them on here in full detail:

image

image

Reducing the time it takes to deploy them is fine, 10 seconds is admittedly excessive, however doubling the available amount goes explicitly against what I outlined in mine.

>  It is intended for the station to prioritize the most important areas and evacuate crew into shielded areas.

🔑

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -1114,7 +1114,7 @@
 	name = "Shield Generator Satellite"
 	desc = "Protect the very existence of this station with these Anti-Meteor defenses. Contains seven bluespace capsules which a single unit of Shield Generator Satellite is compressed within each."
 	cost = 7000
-	max_supply = 2
+	max_supply = 4
```
Comment (Rukofamicom) [inline code/modules/cargo/packs.dm:1117]:
```suggestion
	max_supply = 2
```

I will remove my key and approve if this is reverted.

**Review by Rukofamicom — CHANGES_REQUESTED** [review]

**Comment by EvilDragonfiend** [comment]
I agree with Ruko to reduce the time, but not increasing the stock amount. If you want to prepare meteor shielding perfectly, you should have purchased crates, not being capable of buying a lot in a single move.

**Comment by PowerfulBacon** [comment]
The biggest improvement to meteor shields would be to set them up using a prox monitor so they can actually shoot down meteors outside of process()

**Comment by Rukofamicom** [comment]
Been a month with no movement.

**Pull request closed without merging**